public/live-preview-embed-script.js

```
@@ -0,0 +1,210 @@
+      return;
+    } ;    
+    
+    const { type, payload } = event.data;
+
+    switch (type) {
+      case TWEAKCN_MESSAGE.PING:
+        sendMessageToParent({ type: TWEAKCN_MESSAGE.PONG });
+        break;
+
+      case TWEAKCN_MESSAGE.CHECK_SHADCN:
+        const supportInfo = checkShadcnSupport();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)